### PR TITLE
docs: use yarn instead of npm in contribute section

### PR DESCRIPTION
**Pull request by ddanielcruzz**
Since `yarn.lock` was introduced in this commit https://github.com/withfig/autocomplete/commit/aec6acad7d2c286ecf930dc4321c757353ae3b66

I think docs should be updated accordingly 😄

### CODE REVIEW

**Review by mschrage — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,15 +48,17 @@ Use the steps below or follow our getting started guide: [fig.io/docs](https://f
...
 - Download Fig for macOS
-- Node and NPM (or Yarn)
+- Node and Yarn v3
 
 <br/>
 
 **Steps**
 
-1. Click [here](https://GitHub.com/withfig/autocomplete/fork/) to fork this repo.
+1. Make sure you have `yarn` v3 [installed](https://yarnpkg.com/getting-started/install), as that's the package manager used in this repo.
```
Comment (fedeci) [inline README.md:57]:
```suggestion
1. Make sure you have `yarn` [installed](https://classic.yarnpkg.com/lang/en/docs/install), as that's the package manager used in this repo. 
```

File: README.md [hunk 2]
```diff
@@ -48,15 +48,17 @@ Use the steps below or follow our getting started guide: [fig.io/docs](https://f
 **Prerequisites:**
 
 - Download Fig for macOS
-- Node and NPM (or Yarn)
+- Node and Yarn v3
```
Comment (fedeci) [inline README.md:51]:
```suggestion
- Node and Yarn
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Yarn v3 comes bundled with the repo in the `.yarn/` folder, users only need yarn@1 to access it

**Comment by ddanielcruzz** [comment]
> Yarn v3 comes bundled with the repo in the `.yarn/` folder, users only need yarn@1 to access it

Cool, I applied your suggestions.

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Thanks!

**Pull request merged**